content/en/getting_started/synthetics/private_location.md

```diff
@@ -19,19 +19,21 @@ The access to this feature is restricted - if you don't have access, reach out t
 
 ## Overview
 
-Private locations allow you to **monitor internal-facing applications or any private URLs** that aren’t accessible from the public internet. They can also be used to:
+Private locations allow you to **monitor internal-facing applications** or **private URLs** that aren’t accessible from the public internet. 
 
-* **Create custom locations** in areas that are mission-critical to your business.
-* **Verify application performance in your internal CI environment** before you release new features to production with [Synthetic CI test integration][1].
-* **Compare application performance** from both inside and outside your internal network.
+You can also:
```

Comment:
I feel like it makes it slightly less clear that the following use cases are solved by private locations as well - do you think you could rephrase a bit?

Reply:
```suggestion
You can also use private locations to:
```